SU2_CFD/src/solvers/CNSSolver.cpp

```diff
@@ -1034,13 +1034,13 @@ void CNSSolver::Preprocessing(CGeometry *geometry, CSolver **solver_container, C
 
   /*--- Evaluate the vorticity and strain rate magnitude ---*/
 
-  solver_container[FLOW_SOL]->GetNodes()->SetVorticity_StrainMag();
+  GetNodes()->SetVorticity_StrainMag();
 
   StrainMag_Max = 0.0; Omega_Max = 0.0;
   for (iPoint = 0; iPoint < nPoint; iPoint++) {
 
-    StrainMag = solver_container[FLOW_SOL]->GetNodes()->GetStrainMag(iPoint);
-    Vorticity = solver_container[FLOW_SOL]->GetNodes()->GetVorticity(iPoint);
+    StrainMag = GetNodes()->GetStrainMag(iPoint);
+    Vorticity = GetNodes()->GetVorticity(iPoint);
```

Comment:
Good catch here, but not even GetNodes() is needed, just nodes-> would do the trick.
Can you revert changes to this file and CEulerSolver? I'll put these fixes in #861 where these routines were modified.

Reply:
I'll have to change it here, as `solver_container[FLOW_SOL]` is not allocated yet when I call it in the factory methods. I am gonna change it to `nodes` though. Git should not show a conflict if the changes are exactly the same.